pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -295,5 +299,125 @@ func manageOpenShiftControllerManagerDeployment_v311_00_to_latest(client appscli
 	}
 	required.Annotations[util.VersionAnnotation] = os.Getenv("RELEASE_VERSION")
 
+	if cm != nil {
+		globalProxyCAsExist := false
+		for _, v := range required.Spec.Template.Spec.Volumes {
+			if v.Name == "global-proxy-ca" {
+				globalProxyCAsExist = true
+				break
+			}
+		}
+		if !globalProxyCAsExist {
+			cmSource := &corev1.ConfigMapVolumeSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: cm.Name,
+				},
+				Items: []corev1.KeyToPath{
+					{
+						Key:  "ca-bundle.crt",
+						Path: "tls-ca-bundle.pem",
+					},
+				},
+			}
+			required.Spec.Template.Spec.Volumes = append(required.Spec.Template.Spec.Volumes,
+				corev1.Volume{
+					Name: "global-proxy-ca",
+					VolumeSource: corev1.VolumeSource{
+						ConfigMap: cmSource,
+					},
+				},
+			)
+			mount := corev1.VolumeMount{
+				Name:      "global-proxy-ca",
+				MountPath: "/etc/pki/ca-trust/extracted/pem",
```

Comment:
this is the other place you're adding the volume mount.

Reply:
there you go  thanks